requirements.txt

```diff
@@ -1,3 +1,5 @@
 neuroconv
 nwbwidgets
 nwbinspector
+redis
+numpy
```

Comment:
Sorry if it's not clear from the cookiecutter (we probably should make it that way...) for conversion projects we try to use exact version pinning wherever possible as a way to 'freeze' the environment in a configuration that we can guarantee actually works

This would look like

```python
neuroconv==0.3.0
redis==?.?.?  # whichever you're developing on
```

We also usually do this on a subproject-specific level (in requirements files other than the outer most one) since it might change per subproject